aggregator/src/main/java/com/redhat/cloud/notifications/db/EmailAggregationResources.java

```diff
@@ -0,0 +1,61 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import com.redhat.cloud.notifications.models.EmailAggregationKey;
+import org.hibernate.Session;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@ApplicationScoped
+public class EmailAggregationResources {
+
+    @Inject
+    Session session;
+
+    @Transactional
+    public Boolean addEmailAggregation(EmailAggregation aggregation) {
+        try {
+            session.persist(aggregation);
+            session.flush();
```

Comment:
This shouldn't be necessary, the enclosing transaction should flush automatically.